packages/reakit-utils/README.md

```diff
@@ -84,6 +85,18 @@ applyState((value) => value + 1, 1); // 2
 applyState(2, 1); // 2
 ```
 
+### canUseDOM
+
+Returns `true` if it is running in a browser environment or `false` if it is not (SSR).
+
+#### Examples
+
+```javascript
+import { canUseDOM } from "reakit-utils";
+
+const title = canUseDOM ? document.title : "";
+```
+
```

Comment:
You've added it manually, right? Ideally, it should be JSDoc on the `canUseDOM.ts` file. Have you tried that?

Reply:
Oh, nice, will do!